charts/posthog/values.yaml

```diff
@@ -59,7 +61,30 @@ pgbouncer:
   # -- Additional volumes to be added to the pgbouncer deployment
   extraVolumes: []
 
+migrate:
+  # -- Whether to install the PostHog migrate job or not
+  enabled: true
+
+events:
+  # -- Whether to install the PostHog events stack or not
+  enabled: true
+  # -- events horizontal pod autoscaler settings
+  hpa:
+    # -- Boolean to create a HorizontalPodAutoscaler for events stack
+    # -- This experimental
+    enabled: false
+    # -- CPU threshold percent for the events stack
+    cputhreshold: 60
+    # -- Min pods for the events stack
+    minpods: 1
+    # -- Max pods for the events stack
+    maxpods: 10
+  # -- Amount of events pods to run. Ignored if hpa is used
+  replicacount: 1
```

Comment:
non-blocking

I can't see where this is being used

Reply:
Good catch, this is actually a "bug" or a missed implementation (see next commit)